src/Migrations/Version20220120215900.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace CustomerManagementFrameworkBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Pimcore\Migrations\Migration\AbstractPimcoreMigration;
+
+class Version20220120215900 extends AbstractPimcoreMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema): void
+    {
+        $deletionsTable = $schema->getTable('plugin_cmf_deletions');
+
+        if (!$deletionsTable->hasPrimaryKey()) {
+            $this->addSql('ALTER TABLE `plugin_cmf_deletions` ADD PRIMARY KEY (`id`, `entityType`, `type`);');
```

Comment:
Disable `foreign_key_checks` before alter to avoid errors on migrations so something like this https://github.com/pimcore/pimcore/pull/11273/files#diff-125d4e8130bb6598dd2d258980ec5e217198a4b5a3af79a8f6b7658e5a99b176R33

Reply:
@dvesh3 Add it